### PR TITLE
Ngstack 873 bookmarks styling

### PR DESCRIPTION
**Pull request by TTrakostanec**

### CODE REVIEW

File: templates/backoffice/forms/theme.html.twig [hunk 1]
```diff
@@ -0,0 +1,324 @@
+{% extends 'backoffice/layout.html.twig' %}
```
[the remark this answers is not in the document]
Comment (TTrakostanec) [reply templates/backoffice/forms/theme.html.twig:1]: So I can remove all unnecessary widgets (country_widget, content_choice_widget...) ? Is that what you mean? I didcopy/pasted this from IOTA, I didn't know that there are things that will not be needed

[the remark this answers is not in the document]
Comment (emodric) [reply templates/backoffice/forms/theme.html.twig:1]: Yes, most of this is exclusive to IOTA project.

**Comment by TTrakostanec** [comment]
@emodric 
widgets have been removed

**Comment by TTrakostanec** [comment]
@Ljudevit 

I changed a few things to match the notification module because it is inside vendor and they share part of the css

**Review by Ljudevit — APPROVED** [review]
Good from my side.

File: src/Admin/Menu/MainMenuBuilder.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+namespace App\Admin\Menu;
+
+use App\Attribute\AsMenuBuilder;
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use Netgen\Conversations\Participant\ParticipantProviderInterface;
+use Netgen\Conversations\Repository\ConversationRepository;
+
+#[AsMenuBuilder('createMenu', 'app.admin.main_menu')]
+final class MainMenuBuilder
```
Comment (emodric) [inline src/Admin/Menu/MainMenuBuilder.php:14]: Wrong rebase here?

Comment (TTrakostanec) [reply src/Admin/Menu/MainMenuBuilder.php]: Yes, rebase error (this file was moved to `src/Backoffice/Menu`)
It is fixed now.

**Comment by emodric** [comment]
Needs rebase.

**Comment by TTrakostanec** [comment]
@emodric rebase done

**Comment by emodric** [comment]
Thanks @TTrakostanec !

**Pull request merged**